cmd/acra-connector/acra-connector.go

```diff
@@ -228,22 +269,50 @@ func main() {
 			Errorf("Configuration error: can't check is exists AcraConnector private key %v, got error - %v", clientPrivateKey, err)
 		os.Exit(1)
 	}
-	exists, err = utils.FileExists(serverPublicKey)
-	if !exists {
-		log.WithField(logging.FieldKeyEventCode, logging.EventCodeErrorWrongConfiguration).
-			Errorf("Configuration error: AcraServer public key %s doesn't exists", serverPublicKey)
-		os.Exit(1)
+	log.Infof("Client id and client key is OK")
+
+
+	// --------- check public key  -----------
+	if connectorMode == connector_mode.AcraTranslatorMode {
+		translatorPublicKey := fmt.Sprintf("%v%v%v_translator.pub", *keysDir, string(os.PathSeparator), *clientId)
```

Comment:
before we did check public key incorrectly. what about to fix it with new changes where we will use keystore for it and  cal `keystore.GetPeerPublicKey([]byte(*OutgoingServiceId))` to verify that we can load this key and encapsulate knowledge about file naming?

Reply:
updated